### PR TITLE
created landing page of funplay

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created landing page
add two quizes
fp-1

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]

File: landing-page/landing.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import "/reset.css";
+
+
+.main-row> div{
+    border: 1px solid black;
+    box-shadow: 0 1rem 1rem rgba(0,0,0,0.12), 0 1rem 1rem rgba(0,0,0,0.24);
```
Comment (pratik-kumar-sahu) [inline landing-page/landing.css:6]: use var for colors

**Review by pratik-kumar-sahu — APPROVED** [review]
- get rid of uneven spaces.
- Else looks good. Go ahead with merging

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

File: reset.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+/* ---------------------------------------------------common in page */
+.money-heist-btn{
+    background-color: var(--money-heist-primary-color);
+    color: var(--light-color);
+    border-radius: 1rem;
+}
+.quiz-page{
+    width: 80vw;
+    min-height: 80vh;
+    display: flex;
```
Comment (HrishikeshMahalle) [inline reset.css:143]: Its a good practice to use a container and then make  quiz-page use width in percentage

File: reset.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+.skip-btn{
+  border: 2px dashed var(--money-heist-primary-color);
+  background-color: transparent;
+  color: var(--money-heist-primary-color);
+  font-size:2rem;
+}
+.cancel-btn{
+    background-color: var(--money-heist-primary-color);
+    color:beige;
+    font-size: 2rem;
```
Comment (HrishikeshMahalle) [inline reset.css:188]: Hex always a better practice to use Hex

**Review by HrishikeshMahalle — COMMENTED** [review]
All changes looks Gr8. Just have a look at the above suggestion

**Pull request merged**